.changeset/swift-rabbits-appear.md

```diff
@@ -0,0 +1,34 @@
+---
+"@latticexyz/explorer": patch
+---
+
+### Initial Release of Explorer
+
+Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+
+## Getting Started
```

Comment:
With the above framing and next sentence below, we can just remove this headline I think

Reply:
Agreed, removed